### DISCUSSION TITLE
Screen time measures across years?

### DISCUSSION THREAD

**Discussion by carly-gray-19** [now-i-know-my-abcd/docs#77] [Where in the World is…? (Help)] [answered]
I want to be sure I'm interpreting the screen time measures from `nt_y_st` correctly.

For baseline, 1 year, and 2 year, are the summary measures of typical youth screen time the following: `screentime_1_wkdy2_hr`, `screentime_1_wkdy2_min`, `screentime_7_8_wknd_hr`, `screentime_7_8_wknd_min`, `screentime_wkdy_school_hr`, `screentime_wkdy_school_min`, `screentime_wknd_school_hr`, and `screentime_wknd_school_min`?

Years three and four seem to have the more clearly defined summary measures: `stq_y_ss_weekday`, `stq_y_ss_weekday_nm`, `stq_y_ss_weekday_nt`, `stq_y_ss_weekend`, `stq_y_ss_weekend_nm`, `stq_y_ss_weekend_nt`.

Thanks for any help others can provide!

**Comment by angielaird** [comment, marked as answer]
There's a very helpful table on the ABCD Wiki that goes through the iterations of the STQ: [https://wiki.abcdstudy.org/release-notes/non-imaging/novel-technologies.html#screen-time-questionnaire](https://urldefense.com/v3/__https://wiki.abcdstudy.org/release-notes/non-imaging/novel-technologies.html*screen-time-questionnaire__;Iw!!FjuHKAHQs5udqho!LJm0EbWx5lGpkyqMrbDDgUj-Vs5XHpk9Nyf3p_VJz_p91qthCUVqHcJoOgSxPPF_7_hMad28TKJ5oW4x5Pc$)

Between Y1 and Y2, the questionnaire was drastically revised from a likert-reporting style to youth reporting the exact hours and minutes per category. Eventually, individually querying weekday and weekend for every category was reduced to only weekend use then an overarching "how much time overall" which could, theoretically, include multitasking (e.g., texting and watching a show). Unfortunately, because of the differences, new composite summary variables need to be made to make earlier data consistent with later data.

**Reply by carly-gray-19** [reply]
Thank you for your response! I've been working more with these data now and have gotten more familiar with what was asked when and have also manually created new composite summary variables using methods described by [Nivins et al. (2024)](https://doi.org/10.1038%2Fs41598-024-63566-y) and [Bagot et al. (2022)](https://doi.org/10.1016/j.dcn.2022.101150). Anyone who might come across this message is welcome to message me and I can send you my code! Eventually I'll make it publicly available on GitHub but I want to be absolutely sure it's all accurate and well documented.

~~One follow-up question: am I correct in understanding that no questions about TV or video watching were asked at the year 3 follow-up? That's what it seems like based on the documentation you linked and from what I'm seeing in the data, but maybe I've overlooked a question that's worded a little differently at the year 3 follow-up.~~
I just found the # notes in the table you referenced that identify specific variables used to calculate time spent watching TV in years 3 and 4!
